apps/backend/src/utils/compose_parser.py

```
@@ -0,0 +1,378 @@
+                if service_name in services:
+                    service_config = services[service_name]
+                    return service_config.get('image')
+            return None
+        except yaml.YAMLError:
+            return None
+    
+    def find_compose_files_pattern(self) -> List[str]:
+        """Return common Docker Compose file patterns for searching"""
+        return [
+            'docker-compose.yml',
+            'docker-compose.yaml', 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)